### ISSUE TITLE
Themes are not easy to browse

### ISSUE THREAD

**Issue by wiiznokes** [cosmic-utils/tweaks#31]
I would prefer if the themes were using a grid instead of the right bar

**Comment by edfloreshz** [comment]
The reason it was removed from the main view is that it can cause confusion in devices with small screens regarding which themes are actually installed and which aren't. 

I'm thinking a dialog might be better for this, I'm planning to add a search bar as well.

**Comment by wiiznokes** [comment]
Yeah maybe a dialog, but its a shame that only a fraction of the screen is use currently (on large screen)

**Comment by Department-stack** [comment]
I also think that it could be sorted my popularity, recency & alphabetically

**Comment by edfloreshz** [comment]
I'm trying to figure out how to add width to the dialog to create a grid.
